SharedPackages/BrowserServicesKit/Sources/Subscription/Managers/SubscriptionManagerV2.swift

```diff
@@ -430,27 +465,13 @@ public final class DefaultSubscriptionManagerV2: SubscriptionManagerV2 {
         Logger.subscription.debug("Get tokens \(policy.description, privacy: .public)")
 
         do {
-            let currentCachedTokenContainer = try oAuthClient.currentTokenContainer()
-            let currentCachedEntitlements = currentCachedTokenContainer?.decodedAccessToken.subscriptionEntitlements
-
             await migrateAuthV1toAuthV2IfNeeded()
 
             let resultTokenContainer = try await oAuthClient.getTokens(policy: policy)
             let newEntitlements = resultTokenContainer.decodedAccessToken.subscriptionEntitlements
             cachedUserEntitlements = newEntitlements
 
-            // Send "accountDidSignIn" notification when login changes
-            if currentCachedTokenContainer == nil {
```

Comment:
signin-out and entitlementsDidChange notifications removed from here

Reply:
This is good.